tests/Unit/ProtoTests/BasicDiregapic/out/src/Gapic/LibraryGapicClient.php

```diff
@@ -229,8 +229,8 @@ private static function defaultTransport()
         return 'rest';
     }
 
-    /** Implements GapicClientTrait::getSupportedTransports. */
-    private static function getSupportedTransports()
+    /** Implements GapicClientTrait::supportedTransports. */
```

Comment:
How did you catch this?

Reply:
I was reviewing @yash30201 's doc on Storage V2, and how it only will support `grpc`, so I wanted to check what the method was that we'd need to generate (which is this one), and noticed the names were different between here and GAX.

Reply:
...it would be better if we had a test, though.

Reply:
Test added!

https://github.com/googleapis/gapic-generator-php/pull/692/commits/d32d4ba1c6edcd9eaf75ba4ff219bbeb9490aeca